includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -151,10 +230,74 @@ public function render( $args, $url ) {
 				'width'       => $this->args['width'],
 				'height'      => $this->args['height'],
 				'placeholder' => '',
+				'start'       => 0,
 			]
 		);
 
-		if ( empty( $args['video_id'] ) ) {
+		$attributes = array_merge(
+			[ 'data-videoid' => $args['video_id'] ],
+			wp_array_slice_assoc( $args, [ 'layout', 'width', 'height' ] )
+		);
+		if ( ! empty( $args['title'] ) ) {
+			$attributes['title'] = $args['title'];
+		}
+
+		$allowed_data_params = [
+			'cc_lang_pref',
+			'cc_load_policy',
+			'color',
+			'controls',
+			'disablekb',
+			'enablejsapi',
+			'end',
+			'fs',
+			'hl',
+			'iv_load_policy',
+			'list',
+			'listType',
+			'modestbranding',
+			'origin',
+			'playlist',
+			'playsinline',
+			'rel',
+			'widget_referrer',
+		];
+
+		$query_vars  = [];
+		$query_param = wp_parse_url( $url, PHP_URL_QUERY );
+		wp_parse_str( $query_param, $query_vars );
+
+		foreach ( $allowed_data_params as $allowed_data_param ) {
+			if ( isset( $query_vars[ $allowed_data_param ] ) ) {
+				$attributes[ "data-param-$allowed_data_param" ] = $query_vars[ $allowed_data_param ];
+			}
+		}
+
+		foreach ( [ 'autoplay', 'loop' ] as $param ) {
+			if ( isset( $query_vars[ $param ] ) ) {
+				$attributes[ $param ] = $query_vars[ $param ];
+			}
+		}
+
+		if ( ! empty( $args['start'] ) && 0 < intval( $args['start'] ) ) {
+			$attributes['data-param-start'] = intval( $args['start'] );
```

Comment:
It's recommended to use type conversion here, as it's much faster than using `intval()`.

```suggestion
		if ( ! empty( $args['start'] ) && 0 < (int)$args['start'] ) {
			$attributes['data-param-start'] = (int)$args['start'];
```